assets/scss/_variables.scss

```diff
@@ -1,7 +1,7 @@
 // Local docs variables
-$mwd-rosewood:        $rosewood !default;
-$mwd-rosewood-bright: lighten(saturate($mwd-rosewood, 5%), 15%) !default;
-$mwd-rosewood-light:  lighten(saturate($mwd-rosewood, 5%), 45%) !default;
+$mwd-fuschia:        $fuschia !default;
+$mwd-fuschia-bright: lighten(saturate($mwd-fuschia, 5%), 15%) !default;
+$mwd-fuschia-light:  lighten(saturate($mwd-fuschia, 5%), 45%) !default;
```

Comment:
Perhaps these should be `$mwd-primary` so the key isn't churning as frequently as the value

Reply:
Good idea 💡